site/priority.xml

```diff
@@ -27,127 +27,172 @@ limitations under the License.
     <title>Priority Queue Support</title>
   </head>
   <body>
-    <p>
-      RabbitMQ has priority queue implementation in the core as of version <code>3.5.0</code>.
-    </p>
-    <p>
-      You can declare priority queues using the
-      <code>x-max-priority</code> argument. This argument should be an integer
-      indicating the maximum priority the queue should support. For example,
-      using the Java client:
-    </p>
-
-    <pre class="sourcecode java">
+    <doc:section name="intro">
+      <p>
+        RabbitMQ has priority queue implementation in the core as of version <code>3.5.0</code>.
+        Any queue can be turned into a priority one using <a href="/queues.html#optional-arguments">optional arguments</a>.
+        The implementation supports a limited number of priorities.
```

Comment:
Maybe we should mention the range for priorities, as the exact values show up only in the "Using Client-provided Optional Arguments" section, not in the policy one.